Update glossary.rst [doc] (#1026)
avaframe/AvaFrame: docs/glossary.rst

   in particular the Avalanche Modelling Atlas (AMA). Terms are organized in
   alphabetical order, and the terminology builds on the UNESCO Avalanche Atlas
   :cite:`DQ1981`. It also relates to existing guidelines provided by
   avalanche associations and warning services (e.g. the European Avalanche Warning
   Services (EAWS) :cite:`EAWS.2024`, the Canadian Avalanche Association (CAA)
   :cite:`CAA_OGRS2016`, and the American Avalanche Association (AAA) :cite:`AAA_SWAG2016`).
   The glossary follows common conventions used in related publications
   and projects (see our :ref:`zreferences:References`).

.. glossary::
    :sorted:

    alpha point
        The alpha point refers to the furthest point of the runout of an avalanche :term:`event`
        and marks the terminus of the avalanche deposition.

        see also :term:`runout point`

    alpha angle
        The alpha angle is the angle between the :term:`origin point` and the :term:`runout
        point`. It describes the inclination of the line connecting the top of the avalanche release and the
        furthest point of the deposition. The alpha angle is based on the :term:`runout length`

>         and the altitude difference between the :term:`origin point` and the :term:`runout point`.

Reviewer: ```suggestion
        and the altitude difference between the :term:`origin point` and the :term:`runout point` along the :term:`thalweg`.
```

        the :term:`runout angle`.

        see also :term:`runout angle`

    beta point
        The boundary between the :term:`zone of transition` and :term:`zone of deposition` (start of
        deposition) is often referred to as the 'beta point' or :term:`deposition point` and is associated
        with a certain slope threshold of the :term:`thalweg`. For major avalanche :term:`path` s that may produce
        avalanches of :term:`size` 4-5, the beta point is associated with the 10° point
        (or beta_10), i.e. the point where the slope angle of the :term:`thalweg`
        decreases below 10°. For an avalanche :term:`path` with a :term:`size` 1-3 :term:`event`,
        the beta point may accordingly be associated with larger :term:`thalweg`
        slope angles (up to 30°, see :term:`zone of transition`).

        see also :term:`deposition point`

    beta angle
        The beta angle describes the inclination along the :term:`thalweg` between the
        :term:`origin point` and the :term:`deposition point`. The corresponding :term:`travel angle` is
        referred to as 'beta angle'. It is an important concept in avalanche modeling and simulation. 

>         The travel angle increases with decreasing avalanche :term:`size`.

Reviewer: ```suggestion
        The beta angle increases with decreasing avalanche :term:`size`.
```

    coordinate transformation
        Coordinate transformations refer to the operation of changing
        coordinates, e.g. between a fixed, Eularian, global coordinate system
        with geographical orientation, to an avalanche :term:`path` dependent
        coordinate system along the :term:`thalweg`, or even a Langrangian coordinate
        system, moving along particle trajectories.

    projection
        Projection refers to
        the transition of a defined set of coordinates from one coordinate system to another, i.e. the
        projection of the :term:`runout point` (as furthest reach of the avalanche)
        to the :term:`thalweg`. Or projecting a 3D travel length (xyz) to a 2D
        travel length measured only along xy.

    avalanche cycle
        An avalanche cycle describes a series of natural :term:`event` (s) that occur across a region
        over a relatively short time span (hours to days).

    danger scale
        The avalanche danger scale refers to the avalanche hazard and is an
        inherent part of avalanche warning.

    dense flow
        Dense flow is a :term:`form of movement` in the :term:`zone of transition` of the

>         avalanche. Dense Flow Avalanches (DFA) flow along the ground and are

Reviewer: ```suggestion
        avalanche. Dense Flow Avalanches (DFA) flow along the ground
```

>         associated with warm flow. Mixed types of movement are often observed,

Reviewer 2: ```suggestion
        associated with warm flow. Mixed types of movement are often observed,
```

>         associated with warm flow. Mixed types of movement are often observed,

Reviewer: ```suggestion
. Mixed types of movement are often observed,
```

>         associated with warm flow. Mixed types of movement are often observed,

Reviewer: ```suggestion
. Mixed types of movement are often observed,
```

        different zones (i.e. :term:`zone of origin`, :term:`zone of transition`, :term:`zone of deposition`)
        of an avalanche :term:`path` and allow defining other associated properties, such as :term:`alpha angle`
        or :term:`runout length` that are defined in combination with the avalanche :term:`thalweg`.

    release scenario
        One or more release areas, which are associated with a certain avalanche :term:`path` and release a
        the same time.

    flow variables
        Flow variables include the flow :term:`thickness`, flow :term:`velocity`, and flow :term:`density` and are
        determined by the :term:`form of movement`. The spatio-temporal evolution of these variables
        are usually calculated by the implemented flow models. These models output
        the maximum values over the whole flow, or peak values thoughout the computational duration.
        The flow variables are used to derive other variables such as impact pressure or kinetic
        energy of the flow.

    form of movement
        Is an avalanche criterion in the :term:`zone of transition` and can have :term:`dense flow` and/or
        :term:`powder snow` characteristics.

>         #Todo: Is gliding not a form of movement? what is meant by "avalanche criterion"?

Reviewer: ```suggestion

```

        avalanche with :term:`dense flow` component’.

    return period
        The return period is the average time interval between events of a certain intensity or :term:`size`.
        The return period is often determined by the :term:`runout length` of historically documented avalanche :term:`event` (s).
        It is a time-based measure (how often an event occurs).
        Return periods are often used for planning and risk management.

    return level
        The return level is the magnitude or size of an event that is expected to be exceeded with a certain probability
        over a specified time period. It is a magnitude-based measure (the size of the event).
        Return levels are used for designing infrastructure (like dams) to withstand specific events.

    release area
        Potential release areas are located in the :term:`zone of origin`. Each documented
        :term:`event` or simulation :term:`scenario` is associated to one or more primary
        and/or secondary release areas, that can further be described by the :term:`manner of starting`.

    runout area

>         see :term:`zone of deposition`

Reviewer: ```suggestion
       The runout area is associated to a specific avalanche event and usually located in the :term:`zone of deposition`.
```

    runout angle
        The runout angle, also referred to as :term:`alpha angle`. It describes the inclination of the

>         the avalanche event from the :term:`origin point` to the :term:`runout point`.

Reviewer: ```suggestion
        the avalanche event from the point of release or :term:`origin point` to the :term:`runout point`.
```

    beta point
        see :term:`deposition point`

        The boundary between the :term:`zone of transition` and :term:`zone of deposition` (start of
        deposition) is often called 'beta point' or :term:`deposition point` and is associated
        with a certain slope threshold of the :term:`thalweg`. For major avalanche :term:`path` that may produce
        avalanches of :term:`size` 4-5, the beta point is associated with the 10° point
        (or beta_10), i.e. the point where the slope angle of the :term:`thalweg`
        decreases below 10°. For avalanche :term:`path` with :term:`size` 1-3 :term:`event`
      , the beta point may accordingly be associated with larger :term:`thalweg`
        slope angles (up to 30°, see :term:`zone of transition`).

    beta angle
        see :term:`travel angle`

        The beta angle describes the incliniation along the :term:`thalweg` between the 
        :term:`origin point` and the :term:`deposition point`. The corresponding :term:`travel angle` is
        referred to as 'beta angle'. It is an important concept in avalanche modeling and simulation. 

>         The travel angle increases with decreasing avalanche :term:`size`.

Reviewer 3: ?

        coordinates, e.g. between a fixed, Eularian, global coordinate system
        with geographical orientation, to an avalanche :term:`path` dependent
        coordinate system along the :term:`thalweg`, or even a Langrangian coordinate
        system, moving along particle trajectories.

    projection
        Projection refers to
        the transition of a defined set of coordinates from one coordinate system to another, i.e. the
        projection of the :term:`runout point` (as furthest reach of the avalalanche)
        to the :term:`thalweg`. Or projecting a 3D travel length (xyz) to a 2D
        travel length measured only along xy.

    avalanche cycle
        An avalanche cycle describes a series of :term:`event` (s) that occur across a region
        over a relatively short time span.

    danger scale
        The avalanche danger scale refers to the avalanche hazard and is an
        inherent part of avalanche warning.

    dense flow
        Dense flow is a :term:`form of movement` in the :term:`zone of transition` of the
        avalanche. Dense Flow Avalanches (DFA) flow along the ground and are

>         associated with warm flow. Mixed types of movement are often observed,

Reviewer: remove the warm flow comment - it does not fit. rest is correct (i.e. DFA is a form of movement within the transition zone according to Avi Atlas)

        and marks the end of the avalanche deposition.

    alpha angle
        see :term:`runout angle`

        The alpha angle is the angle between the :term:`origin point` and the :term:`runout
        point`. It describes the general inclination between the release of the avalanche to the
        furthest point of the deposition. The alpha angle is based on the :term:`runout length`
        and the altitude difference between the :term:`origin point` and the :term:`runout point`.
        It is an important element in avalanche simulation and modeling (e.g. alpha-beta model,
        FlowPy simulations). The alpha angle is also termed the angle of reach or the :term:`runout angle`.

    beta point
        see :term:`deposition point`

        The boundary between the :term:`transition` and :term:`deposition` zone (start of
        deposition) is often called beta point or :term:`deposition point` and associated
        with a certain slope threshold of the :term:`thalweg`. For major avalanche :term:`path` that may produce
        avalanches of :term:`size` 4-5, the beta point is associated with the 10° Point

>         (or beta_10), i.e., the point where the slope angle of the :term:`thalweg`

Reviewer 4: ```suggestion
        (or beta_10), i.e. the point where the slope angle of the :term:`thalweg`
```

        of an avalanche :term:`path` and allow defining other associated properties, such as :term:`alpha angle`
        or :term:`runout length` that are defined in combination with the avalanche :term:`thalweg`.
        Besides observed and documented avalanche events, design events of particular 
        :term:`return period` (s) are of particular interest for engineering applications.

    flow variables
        Flow variables include the flow :term:`thickness`, flow :term:`velocity`, and flow :term:`density` and are
        determined by the :term:`form of movement`. The spatio-temporal evolution of these variables
        are usually calculated by the implemented flow models. These models output
        the maximum values over the whole flow, or peak values thoughout the computational duration.
        The flow variables are used to derive other variables such as impact pressure or kinetic
        energy of the flow.

    form of movement
        Is an avalanche criterion in the :term:`zone of transition` and can have :term:`dense flow` and/or
        :term:`powder snow` characteristics.

>         #Todo: Is gliding not a form of movement? what is meant by "avalanche criterion"?

Author: @jtfisch : please check

>         #Todo: Is gliding not a form of movement? what is meant by "avalanche criterion"?

Reviewer: @leon-wagner - please double check the Avalanche Atlas wording : 
![image](https://github.com/user-attachments/assets/99cbc61b-2e45-4535-9454-afc5fccd68b4)

    manner of starting
        Is an avalanche criterion in the :term:`zone of origin` and has the
        characteristics 'loose', 'slab', or 'gliding'.

    terrain classification

>         Terrain may be classified according to the Avalanche Terrain Exposure

Reviewer 3: something like: "Terrain classification refers to the systematic categorization of different types of ground surfaces based on their physical/optical/... or other characteristics."

        angle difference of the slope and the normal.

    travel angle

>         The travel angle describes the general inclination between the :term:`origin point` and the current location of interest. 

Reviewer 4: ```suggestion
        The travel angle describes the inclination between the :term:`origin point` and the current location of interest. 
```

    travel length
        Travel lengths are measured as horizontally projected travel length
        (:math:`s_{XY}`) along the :term:`thalweg`, measured between the current
        location with the uppermost point of the release, :term:`origin point`.
        Alternatively, the surface parallel travel length (:math:`s_{XYZ}`) may be
        defined as the three-dimensional length travelled by the avalanche.

    path
        The avalanche path summarizes the total catchment and is divided into
        different zones (:term:`zone of origin`, :term:`zone of transition`, :term:`zone of deposition`)
        with different criteria and characteristics. An inherent property of the avalanche path is the
        :term:`thalweg` and the associated avalanche :term:`event`.

    powder snow
        Powder snow avalanches (PSA) refer to the :term:`form of movement` in the 
        :term:`zone of transition`, referring to the dust or suspension cloud in avalanches.
        PSA are associated with cold, dry cohesionless snow. Mixed types of
        movement are often observed, combining different flow regimes and their

>         transitions, e.g., ‘powder avalanche with flow component’.

Reviewer 4: ```suggestion
        transitions, e.g. ‘powder avalanche with flow component’.
```

        estimates, which are associated to the occurrence probability.

    release area
        Potential release areas are located in the :term:`zone of origin`. Each documented
        :term:`event` or simulation :term:`scenario` is associated to one or more primary
        and/or secondary release areas, that can further be described by the :term:`manner of starting`.

    runout area
        see :term:`zone of deposition`

    runout angle
        The runout angle, also referred to as :term:`alpha angle`, describes the general inclination of the
        the avalanche event from the :term:`origin point` to the :term:`runout point`.

    runout length

>         Runout lengths is intricately linked to the :term:`alpha point`, utilizing the

Reviewer 4: ```suggestion
        Runout lengths is intrinsically linked to the :term:`alpha point`, utilizing the
```

        the uppermost point of the :term:`release area` (:term:`origin point`) to furthest reach of the runout
        area (:term:`runout point`).

    runout point
        The runout point is also referred to as :term:`alpha point`. It describes the furthest point of the runout,
        and marks the outer most end of the avalanche deposit. The runout may refer to visible deposition
        (associated to dense flow), damages or the impacted and affected area (associated to air
        blast or :term:`powder snow`) in the zone of :term:`deposition` and is usually defined
        via flow :term:`thickness`, velocity, kinetic energy or impact pressure thresholds.

    destructive size
        The destructive size refers to the magnitude or intensity of an :term:`event`, according to the EAWS size classification, which is closely
        related to the CAA destructive size. Thereby the size refers to the destructive potential, :term:`runout length`,  volumn or mass 
        of each avalanche event. 

    relative size
        The relative size refers to the size of an avalanche in relation to the :term:`thalweg`. The classification scheme was developed 

>         by the AAA and relys on the horizontal extent, vertical depth of the fracture, volumn and mass of the deposited snow as well as the

Reviewer 4: ```suggestion
        by the AAA and relies on the horizontal extent, vertical depth of the fracture, volume and mass of the deposited snow as well as the
```

>         runout length of the avalanche event. Thereby the largest possible event along a :term:`thalweg` is classified with R5.

Reviewer 4: ```suggestion
        runout length of the avalanche event. Therefore the largest possible event along a :term:`thalweg` is classified with R5.
```

    Dmax
        Dmax is the size of the maximum potential avalanche event size in an infinite time series of avalanches 

>         within the terrain of a given avalanche path. This maximum potential avalanche event corresponds with the 

Reviewer 4: ```suggestion
        within the terrain of a given avalanche path. This maximum potential avalanche event corresponds to the 
```

        relative size R5 (major or maximum, relative to path) of the relative avalanche size classification 

>         (American Avalanche Association, 2016). Therefore Dmax is defined accordingly to the destructive size

Reviewer 4: ```suggestion
        (American Avalanche Association, 2016). Therefore Dmax is defined according to the destructive size
```

>         (accoridng to the avalanche size classifications of the Canadian Avalanche Association and the European

Reviewer 4: ```suggestion
        (see avalanche size classifications of the Canadian Avalanche Association and the European
```

        The origin point describes the highest possible release point along a :term:`thalweg` within the terrain.

    zone of origin
        The zone of origin delineates the area, in which typical :term:`release area` (s) are located, and an avalanche's
        appearance is characterized by the :term:`manner of starting`. The uppermost
        possible point is referred to as :term:`origin point`.

    transition point
        see also :term:`zone of transition`

        The transition point describes the transition between the zone of origin and the zone of transition.
        It marks the lowest point of the release area. It is generally assumed that the transition occurs at
        a slope angle of approximately 30°.

    zone of transition
        The zone of transition is the area between :term:`zone of origin` and
        :term:`zone of deposition` along the :term:`thalweg`. The :term:`form
        of movement` is linked to the :term:`flow variables`. The start of
        transition links the :term:`zone of origin` and transition and is usually
        associated with a slope inclination of about 28-30°.

    deposition point
        see also :term:`zone of deposition`

>         The deposition point describes the point along the thalweg where an avalanche event starts to decelerates

Reviewer 4: ```suggestion
        The deposition point describes the point along the thalweg where an avalanche event starts to decelerate
```
